vector/src/main/java/im/vector/adapters/VectorParticipantsAdapter.java

```diff
@@ -878,12 +879,9 @@ public View getGroupView(final int groupPosition, final boolean isExpanded, View
         }
 
         if (groupPosition != mKnownContactsSectionPosition || mParticipantsListsList.get(groupPosition).size() > 0) {
-            if (isExpanded) {
-                imageView.setImageResource(R.drawable.ic_material_expand_less_black);
-            } else {
-                imageView.setImageResource(R.drawable.ic_material_expand_more_black);
-            }
 
+            int expandLogoRes = isExpanded ? R.drawable.ic_material_expand_more_black : R.drawable.ic_material_expand_less_black;
```

Comment:
I see that you added some consistency here, because if else block have been swapped.
Ok for consistency, but do you think it's better this way?

Reply:
Yes ! there is not huge work done here in if else block, thats why used ternary operator for this simple work